doc/rtt.js

```diff
@@ -0,0 +1,178 @@
+ *              RTT.stop(function(err) {
+ *                  console.log('Stopped');
+ *              });
+ *          });
+ *      });
+ * });
+ *
+ * @module pc-nrfjprog-js.RTT
+ */
+
+/**
+ * Possible error.<br/>
+ * If an operation completed sucessfully, the error passed to the callback
+ * function will be <tt>undefined</tt> (and thus, falsy).<br/>
+ * This will be an instance of the built-in {@link https://nodejs.org/dist/latest/docs/api/errors.html#errors_class_error|Error} class, with some extra properties:
+ * @typedef {Error} Error
+ * @property {integer} errno
+ *    The error number. Value will be one of the following predefined constants:<br/>
+ *    <tt>nrfjprogjs.RTTCouldNotLoadHighlevelLibrary</tt><br/>
+ *    <tt>nrfjprogjs.RTTCouldNotOpenHighlevelLibrary</tt><br/>
+ *    <tt>nrfjprogjs.RTTCouldNotGetDeviceInformation</tt><br/>
+ *    <tt>nrfjprogjs.RTTCouldNotLoadnRFjprogLibrary</tt><br/>
+ *    <tt>nrfjprogjs.RTTCouldNotOpennRFjprogLibrary</tt><br/>
+ *    <tt>nrfjprogjs.RTTCouldNotConnectToDevice</tt><br/>
+ *    <tt>nrfjprogjs.RTTCouldNotStartRTT</tt><br/>
+ *    <tt>nrfjprogjs.RTTCouldNotFindControlBlock</tt><br/>
+ *    <tt>nrfjprogjs.RTTCouldNotGetChannelInformation</tt><br/>
+ *    <tt>nrfjprogjs.RTTCouldNotCallFunction</tt><br/>
+ *    <tt>nrfjprogjs.RTTNotInitialized</tt><br/>
+ *    <tt>nrfjprogjs.RTTCouldNotExecuteDueToLoad</tt><br/>
+ *
+ * @property {String} errcode A human-readable version of the error code.
+ * @property {String} erroroperation The internal function that caused the error.
+ * @property {String} errmsg Error string. The value will be equal to that of the built-in <tt>message</tt> property.
+ * @property {integer} lowlevelErrorNo The low-level error code, if applicable.
+ * @property {String} lowlevelError A human-readable version of the low-level error code.
+ * @property {String} log The complete log from the internal functions.
+ *
+ * @example
+ * RTT0.start(12345678, {}, function(err, down, up){
+ *     if (err) {
+ *         throw err;
+ *     } else {
+ *         // success
+ *     }
+ * });
+ *
+ * @example
+ * RTT.start(12345678, {}, function(err, down, up){
+ *     if (err && err.errno === nrfjprogjs.RTTCouldNotFindControlBlock) {
+ *          console.error('The firmware is not RTT capable');
+ *     }
+ * });
+ */
+
+/**
+ * Information about the different up and down channels available on the device
+ * @typedef ChannelInfo
+ * @property {integer} channelIndex The index used to address this channel
+ * @property {String} name The name of the channel
+ * @property {integer} size The size of the channel
+ */
+
+ /**
+ * Option flags to be used when starting RTT. This may speed up the process of locating the control block, and the RTT Start.
+ * @typedef StartOptions
+ * @property {integer} controlBlockLocation The location of the control block. If this location is not the start of the RTT control block, start will fail.
```

Comment:
I'm pretty sure this is not an `integer`

Reply:
It should be an int. Will check.